### ISSUE TITLE
Option to hide Location in the media toolbar

### ISSUE THREAD

**Issue by danieltigse** [mesibo/messenger-app-android#16]
Is there a way to hide the location button? If you make the MesiboUI repository as open source I could help you guys 👍

**Comment by mesibo** [comment]
Yes,  it will be eventually open source. In the meantime, I have taken your request to the development team.

**Comment by danieltigse** [comment]
Thanks! I appreciate it! Any update of this?

**Comment by danieltigse** [comment]
Hi. Do you have any update regarding this request?
